### PR TITLE
Fix Debian Buster repo

### PR DESCRIPTION
**Pull request by rudysp**
Specific Docker 19.03.x version installer script for Debian Buster didn't have the correct sources.list repo update.

### CODE REVIEW

**Comment by superseb** [comment]
Does this not work on Debian buster today? Can you share the error?

**Comment by rudysp** [comment]
> 
> 
> Does this not work on Debian buster today? Can you share the error?
+ + sh -c apt-key add -
curl -fsSl https://download.docker.com/linux/debian/gpg
OK
+ sh -c add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/debian 10 stable"
+ [ debian = debian ]
+ [ 10 = wheezy ]
+ sh -c apt-get update
Hit:1 http://security.debian.org/debian-security buster/updates InRelease
Hit:2 http://kebo.pens.ac.id/debian buster InRelease
Hit:3 http://kebo.pens.ac.id/debian buster-updates InRelease
Ign:4 https://download.docker.com/linux/debian 10 InRelease
Err:5 https://download.docker.com/linux/debian 10 Release
  404  Not Found [IP: 13.225.2.103 443]
Reading package lists... Done
E: The repository 'https://download.docker.com/linux/debian 10 Release' does not have a Release file.

@superseb looks like my fresh Debian Buster doesn't have lsb-release package just yet, thus, function check_forked() skipped. But run it twice will solve the issue as software-properties-common installation will include lsb-release. Or, just add "buster" in dist_version selection option.

**Comment by superseb** [comment]
OK, makes sense to add it, can you update to include `19.03.14.sh`? Also, are the other version scripts not affected?

**Comment by rudysp** [comment]
Other versions are affected as well, ok i'll update all 19.03.x and 18.09.x versions.

**Comment by superseb** [comment]
Thanks, please squash commits to one

**Comment by rudysp** [comment]
I'm not sure whether I've done the correct squash process or not, please help to check.

File: 18.09.0.sh [hunk 1]
```diff
@@ -336,6 +336,9 @@ do_install() {
 		debian|raspbian)
 			dist_version="$(cat /etc/debian_version | sed 's/\/.*//' | sed 's/\..*//')"
 			case "$dist_version" in
+			    10)
```
Comment (superseb) [inline 18.09.0.sh:339]: This seems out of line

**Comment by superseb** [comment]
It should show 1 commit in the PR tab Commits, here is a description of squashing commits into a single one: https://www.devroom.io/2011/07/05/git-squash-your-latests-commits-into-one/

**Comment by rudysp** [comment]
I did exactly like the article did, don't know what missing, this is new to me though. I will close this pull request and make another cleaner pull.

**Pull request closed without merging**